homeassistant/components/devolo_home_network/strings.json

```diff
@@ -6,11 +6,17 @@
         "description": "[%key:common::config_flow::description::confirm_setup%]",
         "data": {
           "ip_address": "[%key:common::config_flow::data::ip%]"
+        },
+        "data_description": {
+          "ip_address": "IP address of the device that will be set up."
         }
       },
       "reauth_confirm": {
         "data": {
           "password": "[%key:common::config_flow::data::password%]"
+        },
+        "data_description": {
+          "password": "Password the device is protected with."
```

Comment:
Is there only 1 password? How does one know their password?

Reply:
If there is a password (which is not mandatoy), the user set it itself using the device's website. So I could also refrase it to

"Password you protected the device with."